src/OSPSuite.Core/Serialization/Xml/PKMLPersistor.cs

```diff
@@ -56,16 +57,30 @@ public PKMLPersistor(
          _cloneManagerForModel = cloneManagerForModel;
       }
 
+      public string Serialize<T>(T entityToSerialize)
+      {
+         using (var serializationContext = SerializationTransaction.Create(_container))
+         {
+            return xElementFor(entityToSerialize, serializationContext).ToString();
+         }
+      }
+
```

Comment:
I also needed a way to serialize building blocks to PKML without writing to file so that they can be shared with MoBi.